mungegithub/submit-queue/deployment/kubernetes/configmap.yaml

```diff
@@ -16,10 +16,10 @@ data:
 
   # submit-queue options.
   submit-queue.required-contexts: "Jenkins GCE Node e2e"
-  submit-queue.nonblocking-jenkins-jobs: kubernetes-e2e-gke-staging,kubernetes-e2e-gke-staging-parallel,kubernetes-e2e-gce-serial,kubernetes-e2e-gke-serial,kubernetes-e2e-gke-test,kubernetes-e2e-gce-examples,kubernetes-e2e-gce-federation,kubernetes-e2e-gce-scalability,kubernetes-soak-continuous-e2e-gce,kubernetes-soak-continuous-e2e-gke
-  submit-queue.jenkins-jobs: kubelet-gce-e2e-ci,kubernetes-build,kubernetes-test-go,kubernetes-verify-master,kubernetes-e2e-gce,kubernetes-e2e-gce-slow,kubernetes-e2e-gke,kubernetes-e2e-gke-slow,kubernetes-kubemark-5-gce
+  submit-queue.nonblocking-jenkins-jobs: kubernetes-e2e-gke-staging,kubernetes-e2e-gke-staging-parallel,kubernetes-e2e-gce-serial,kubernetes-e2e-gke-serial,kubernetes-e2e-gke-test,kubernetes-e2e-gce-examples,kubernetes-e2e-gce-federation,kubernetes-e2e-gce-scalability,kubernetes-soak-continuous-e2e-gce,kubernetes-soak-continuous-e2e-gke,kubernetes-kubemark-5-gce,kubernetes-kubemark-500-gce
+  submit-queue.jenkins-jobs: kubelet-gce-e2e-ci,kubernetes-build,kubernetes-test-go,kubernetes-verify-master,kubernetes-e2e-gce,kubernetes-e2e-gce-slow,kubernetes-e2e-gke,kubernetes-e2e-gke-slow
   submit-queue.presubmit-jobs: kubernetes-pull-build-test-e2e-gce
-  submit-queue.weak-stable-jobs: kubernetes-kubemark-500-gce
+  submit-queue.weak-stable-jobs:
```

Comment:
Updated, PTAL

On Mon, Aug 1, 2016 at 5:12 PM, Anirudh Ramanathan <notifications@github.com
> wrote:

> In mungegithub/submit-queue/deployment/kubernetes/configmap.yaml
> <https://github.com/kubernetes/contrib/pull/1461#discussion_r73075916>:
>
> >    submit-queue.presubmit-jobs: kubernetes-pull-build-test-e2e-gce
> > -  submit-queue.weak-stable-jobs: kubernetes-kubemark-500-gce
> > +  submit-queue.weak-stable-jobs:
>
> Previously, when we weren't using the configmap, we'd just remove the
> argument entirely and that would work (because of the default set in code
> to []string{}. We can't do that anymore since we're populating
> environment variables with these configmap values. We should be able to
> once we consume the configmap directly in the pod (#1415
> <https://github.com/kubernetes/contrib/issues/1415>), instead of
> supplying commandline parameters.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/contrib/pull/1461/files/b11e5e6381f23c40d67f45fced292036aa0baf71#r73075916>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAngljau_kRxlI8srW87OSig9qM1Sf69ks5qbotwgaJpZM4JaHDj>
> .
>